src/lib/keypom.ts

```diff
@@ -109,6 +109,15 @@ class KeypomJS {
     return keyInfo.remaining_uses;
   };
 
+  checkIfDropExists = async (secretKey: string) => {
+    try {
```

Comment:
can be simplified to:

```
checkIfDropExists = async (secretKey: string) => await getDropInformation({ secretKey }).then(() => true).catch((_) => false);
```

Reply:
hmm i prefer to use try catch and avoid promise chaining. I feel its more readable

Reply:
and normally when a Promise is awaited, we should use catch to contain the error

Reply:
https://stackoverflow.com/questions/54385676/whats-wrong-with-awaiting-a-promise-chain

Reply:
noted! ok let's go with this!